docs/samples/python/presidio_notebook.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
I've seen notebooks starting with pip install, do we want to add it?

Reply:
I'm assuming the analyzer and anonymizer are already installed when running this.